packages/eslint-plugin/tests/rules/adjacent-overload-signatures.test.ts

```diff
@@ -256,6 +256,24 @@ class Test {
   invalid: [
     {
       code: `
+function wrap() {
+  function foo(s: string);
+  function foo(n: number);
+  export type bar = number;
```

Comment:
This specific `export` line isn't valid syntax, since it's inside a function:
https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABAdwE4EMAOAKAlIgbwChFFRJYEy45sBnALkTqlRjAHNcBuEs8aPCTAa2MEzAgAtgCMApqh585AD0xxUURFACemOYhnpUiALyJJshb1LlBVEbTrjmrdh0QAfC9PmLCAL5EQUA

```suggestion
```

Reply:
You're right! Since it's an invalid case, I'll only delete the `export` keyword.